server/dive_server/viame.py

```diff
@@ -530,10 +568,16 @@ def save_attributes(self, folder, attributes):
         )
     )
     def get_valid_images(self, folder):
-        images = Folder().childItems(
-            getCloneRoot(self.getCurrentUser(), folder),
-            filters={"lowerName": {"$regex": safeImageRegex}},
-        )
+        if fromMeta(folder, SingleMultiCamMarker) in TRUTHY_META_VALUES:
+            images = Folder().childItems(
+                folder,
+                filters={"lowerName": {"$regex": safeImageRegex}},
+            )
+        else:
+            images = Folder().childItems(
+                getCloneRoot(self.getCurrentUser(), folder),
+                filters={"lowerName": {"$regex": safeImageRegex}},
+            )
```

Comment:
It might be better to handle the `SingleMultiCamMarker` case from inside `getCloneRoot` intead of having to remember to short-circuit it everytime getCloneRoot is used.

Can be done later, just making a note to future self.